scribus/langmgr.cpp

```diff
@@ -89,90 +89,541 @@ void LanguageManager::generateLangList()
+	m_langTable.append(LangDef("kaa",      "",         "Karakalpak",                          QObject::tr("Karakalpak")));
+	m_langTable.append(LangDef("krl",      "",         "Karelian",                            QObject::tr("Karelian")));
+	m_langTable.append(LangDef("kar",      "",         "Karen",                               QObject::tr("Karen")));
+	m_langTable.append(LangDef("kfr",      "",         "Kachchi",                             QObject::tr("Kachchi")));
+	m_langTable.append(LangDef("ks",       "",         "Kashmiri",                            QObject::tr("Kashmiri")));
+	m_langTable.append(LangDef("csb",      "",         "Kashubian",                           QObject::tr("Kashubian")));
+	m_langTable.append(LangDef("kyu",      "",         "Kayah",                               QObject::tr("Kayah")));
+	m_langTable.append(LangDef("kk",       "",         "Kazakh",                              QObject::tr("Kazakh")));
+	m_langTable.append(LangDef("kjh",      "",         "Khakass",                             QObject::tr("Khakass")));
+	m_langTable.append(LangDef("kht",      "",         "Khamti",                              QObject::tr("Khamti")));
+	m_langTable.append(LangDef("kha",      "",         "Khasi",                               QObject::tr("Khasi")));
+	m_langTable.append(LangDef("km",       "",         "Khmer",                               QObject::tr("Khmer")));
+	m_langTable.append(LangDef("khw",      "",         "Khowar",                              QObject::tr("Khowar")));
+	m_langTable.append(LangDef("ki",       "",         "Kikuyu",                              QObject::tr("Kikuyu")));
+	m_langTable.append(LangDef("sjd",      "",         "Kildin Sami",                         QObject::tr("Kildin Sami")));
+	m_langTable.append(LangDef("kmb",      "",         "Kimbundu",                            QObject::tr("Kimbundu")));
+	m_langTable.append(LangDef("rw",       "",         "Kinyarwanda",                         QObject::tr("Kinyarwanda")));
+	m_langTable.append(LangDef("mwk",      "",         "Kita Maninkakan",                     QObject::tr("Kita Maninkakan")));
+	m_langTable.append(LangDef("kex",      "",         "Kituba (Congo)",                      QObject::tr("Kituba (Congo)")));
+	m_langTable.append(LangDef("kex",      "",         "Kokni",                               QObject::tr("Kokni")));
+	m_langTable.append(LangDef("ksh",      "",         "Kölsch",                              QObject::tr("Kölsch")));
+	m_langTable.append(LangDef("kv",       "",         "Komi",                                QObject::tr("Komi")));
+	m_langTable.append(LangDef("koi",      "",         "Komi-Permyak",                        QObject::tr("Komi-Permyak")));
+	m_langTable.append(LangDef("kpv",      "",         "Komi-Zyrian",                         QObject::tr("Komi-Zyrian")));
+	m_langTable.append(LangDef("kmw",      "",         "Komo (Democratic Republic of Congo)", QObject::tr("Komo (Democratic Republic of Congo)")));
```

Comment:
I think “Congo” instead of the long “Democratic Republic of Congo” should be enough, as done above already.